cmd/erasure-server-pool-decom.go

```diff
@@ -821,6 +817,22 @@ func (z *erasureServerPools) decommissionPool(ctx context.Context, idx int, pool
 				var failure, ignore bool
 				// gr.Close() is ensured by decommissionObject().
 				for try := 0; try < 3; try++ {
+					stopFn := globalDecommissionMetrics.log(decomMetricDecommissionObject, idx, bi.Name, version.Name, version.VersionID)
```

Comment:
Previously `stopFn`, the function which computes the time elapsed during a `z.DecommissionObject` call in addition to publishing the relevant trace event, encloses only `z.DecommissionObject`. With this change, for non-transitioned objects, it encloses both `z.GetObjectNInfo` and `z.DecommissionObject`. We need to define `stopFn` at different point in time for transitioned and non-transitioned objects.